account_due_list/__manifest__.py

```diff
@@ -18,11 +18,12 @@
     'author': 'Odoo Community Association (OCA), '
               'Agile Business Group, '
               'Tecnativa,'
-              'Zikzakmedia SL',
+              'Zikzakmedia SL, '
+              'Ursa Information Systems',
```

Comment:
Adding Ursa as author changing only the dependency is not a meaningful co-authoring reason I'm afraid.